LiteCore/Storage/SQLiteKeyStore.cc

```diff
@@ -353,7 +353,8 @@ namespace litecore {
     }
 
 
-    void SQLiteKeyStore::createIndex(slice expression,
+    void SQLiteKeyStore::createIndex(slice indexName,
```

Comment:
This is a good point to sanity-check the indexName. At a minimum, it shouldn't be empty and shouldn't contain a `"`.